### PR TITLE
feat(sandbox): backend example of payment-service

### LINKED ISSUE #401: Example of Payment-Service to run as a service
Payment-service example run as a backend service

### PR DESCRIPTION
**Pull request by sumiter92**
## Description

Adds example of payment-service to run as a service

Fixes #401 

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] Code conforms with the style guide

### CODE REVIEW

File: sandbox/docker-compose.yml [hunk 1]
```diff
@@ -499,6 +513,21 @@ services:
       - postgres_orchestrator
     restart: on-failure
 
+  payment-backend-example:
```
Comment (akshatdubeysf) [inline sandbox/docker-compose.yml:516]: migrations?

File: sandbox/payment-example/backend/package.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
+{
+  "name": "payment-example-backend",
```
Comment (akshatdubeysf) [inline sandbox/payment-example/backend/package.json:2]: @sourceloop/payment-example-backend

File: sandbox/payment-example/backend/src/application.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import {ServiceMixin} from '@loopback/service-proxy';
+import {PaymentServiceComponent} from '@sourceloop/payment-service';
+import {
+  GatewayBindings,
+  GatewayProvider,
+  RazorpayBindings,
+  RazorpayProvider,
+  StripeBindings,
+  StripeProvider,
+} from '@sourceloop/payment-service/dist/providers';
```
Comment (akshatdubeysf) [inline sandbox/payment-example/backend/src/application.ts:18]: Export these from the index.ts of the service so the import from dist is not required

File: sandbox/payment-example/backend/src/application.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    this.static('/', path.join(__dirname, '../public'));
+
+    // Customize @loopback/rest-explorer configuration here
+    this.configure(RestExplorerBindings.COMPONENT).to({
+      path: '/explorer',
+    });
+    this.component(RestExplorerComponent);
+    this.component(PaymentServiceComponent);
+    this.bind(StripeBindings.Config).to({dataKey: '', publishKey: ''});
+    this.bind(StripeBindings.StripeHelper).toProvider(StripeProvider);
```
Comment (akshatdubeysf) [inline sandbox/payment-example/backend/src/application.ts:43]: These should be bound by default in the component?

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: sandbox/payment-example/backend/package.json [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  "version": "0.0.1",
+  "description": "backend",
```
Comment (akshatdubeysf) [inline sandbox/payment-example/backend/package.json:4]: ??

**Review by akshatdubeysf — CHANGES_REQUESTED** [review]

File: sandbox/payment-example/backend/package.json (hunk 5 above)
Comment (sumiter92) [reply sandbox/payment-example/backend/package.json]: done, changed

File: sandbox/payment-example/backend/package.json (hunk 2 above)
Comment (sumiter92) [reply sandbox/payment-example/backend/package.json]: done, changed

File: sandbox/docker-compose.yml (hunk 1 above)
Comment (sumiter92) [reply sandbox/docker-compose.yml:573]: added

File: sandbox/payment-example/backend/src/application.ts (hunk 3 above)
Comment (sumiter92) [reply sandbox/payment-example/backend/src/application.ts]: done in service and modified here too

File: sandbox/payment-example/backend/src/application.ts (hunk 4 above)
Comment (sumiter92) [reply sandbox/payment-example/backend/src/application.ts]: done

**Pull request merged**